db/migrate/20170901055020_add_password_digest_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPasswordDigestToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :password_digest, :string
+  end
+end
```

Comment:
@erose357 @jtruong2 @swdonovan @markyv18 Remember to run `bundle exec rake db:migrate` when we pull down from master